cmd/rac/cli.go

```diff
@@ -23,9 +23,18 @@ func myUsage() {
 	fmt.Println("Extracts information from Innosat-MATS rac-files")
 	fmt.Println()
 	fmt.Printf("Usage: %s [OPTIONS] rac-file ...\n", os.Args[0])
-	fmt.Println()
 	flag.PrintDefaults()
-	fmt.Println()
+	fmt.Println(`
+The tool can be used to scan rac files for contents use the -stdout flag and
```

Comment:
This sentence seems to be missing a period or a comma after `contents`. Perhaps change the period after `information` to `, e.g.:` as well? Should probably be a `:` after `unlucky` as well.

Reply:
(I think I would prefer `contents use` -> `contents. Use`.)

Reply:
Do you mind creating a suggestion (using the [+-] button) ?

Reply:
If I can find it...